io/www/src/fragment/replace.js

```diff
@@ -76,14 +76,14 @@ function replaceValues(input, dictionary, calls) {
 }
 
 async function replace(context) {
-    const { body } = context;
-    let fieldsString = JSON.stringify(body.fields);
-    if (fieldsString.match(PH_REGEXP)) {
+    let body = context.body;
+    let bodyString = JSON.stringify(body);
+    if (bodyString.match(PH_REGEXP)) {
         const dictionary = await getDictionary(context);
         if (dictionary && Object.keys(dictionary).length > 0) {
-            fieldsString = replaceValues(fieldsString, dictionary, []);
+            bodyString = replaceValues(bodyString, dictionary, []);
             try {
-                body.fields = JSON.parse(fieldsString);
+                body = JSON.parse(bodyString);
```

Comment:
Why changing this here? as you already know i'm not a huge fan in mixing up fixes for history management. And typically here i'm not sure: do we need PH management outside of fields inputs? what is the use case?

Reply:
yes, in the references, there are underneath fields object.